controllers/testdata/declarative/case_2/expected/created/sample-config.yaml.tmpl

```diff
@@ -9,10 +9,4 @@ metadata:
         component: data-science-pipelines
 data:
     sample_config.json: |-
-        [
-          {
-            "name": "[Demo] iris-training",
-            "description": "[source code](https://github.com/opendatahub-io/data-science-pipelines/tree/master/samples/iris-sklearn) A simple pipeline to demonstrate a basic ML Training workflow",
-            "file": "/samples/iris-pipeline-compiled.yaml"
-          }
-        ]
+        {"loadSamplesOnRestart":true,"pipelines":[{"description":"[source code](https://github.com/opendatahub-io/data-science-pipelines/tree/master/samples/iris-sklearn) A simple pipeline to demonstrate a basic ML Training workflow","file":"/samples/iris-pipeline-compiled.yaml","name":"[Demo] iris-training","versionDescription":"","versionName":"[Demo] iris-training - v0.0.0"}]}
```

Comment:
It'd be nice to have a test case where the platform version isn't the default value.